scripts/gen-api-docs.sh

```diff
@@ -35,4 +35,4 @@ do
 done
 
 # Hacky way to get through the "plain text html not allowed" hugo error
-find $OUTPUTDIR -type f -exec sed -i '' '1 s/^/---\n---\n/' {} \;
+find $OUTPUTDIR -type f -exec sed -i '1 s/^/---\n---\n/' {} \;
```

Comment:
Ah, did we do this instead of turning on full HTML rendering in Hugo?

Reply:
We need both, or else it doesn't work.